distributed_social_network/api/views/inbox.py

```diff
@@ -286,6 +286,9 @@ def add_comment(request, author_id, inbox):
 # Returns None if unable to create author
 # This will be used to create local copies of remote authors
 def find_or_create_author(id):
+    if "http://" not in id:
```

Comment:
I think we will want a reusable way to do this, because all the of the views will want to be able to handle different remote nodes

Reply:
This solution will work for any remote node that either has ids as urls, or ids as uuids. Being that team 10 hasnt shown any documentation or evidence of how they will approach this, I don't know what to implement for them. But this works for tiktaktoe.